h_periodic/h_beat.py

```diff
@@ -55,7 +55,7 @@
         "fill-annotations-pk": {
             "options": {"expires": 30},
             "task": "h.tasks.annotations.fill_pk_and_user_id",
-            "schedule": crontab(hour="*,9-12", minute="*/15"),
+            "schedule": crontab(hour="9-12", minute="*/15"),
```

Comment:
My ideal plan was to avoid high load times. I don't think it matter that much now that has been running for a while let's stick to the plan.

Reply:
I guess the thing here is that `*,9-12` is the same as just `*`, when the intention was for it to be `9-12`, but correct me if I'm wrong Marcos

Reply:
Yeah my plan was:

https://crontab.guru/#*/15_9-12_*_*_* 

`At every 15th minute past every hour from 9 through 12.”`

but I had:

https://crontab.guru/#*/15_*,9-12_*_*_*

`“At every 15th minute past every hour and every hour from 9 through 12.”`